src/ids.rs

```diff
@@ -171,9 +166,9 @@ struct CairoStruct {
 }
 
 #[pyclass(unsendable)]
-struct PyTypedId {
+pub struct PyTypedId {
```

Comment:
Could this be pub(crate) instead?

Reply:
Resolved: https://github.com/lambdaclass/cairo-rs-py/pull/145/commits/85619ee14fab9fb14a961ee1c3425d34c5f09e45